cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -194,6 +194,15 @@ spec:
     singular: cephblockpool
   scope: Namespaced
   version: v1
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            replicated:
+              properties:
+                size:
+                  type: integer
```

Comment:
A few thoughts:
- Let's add a `minimum: 0`. We need a minimum of 0 to allow for it being unspecified when erasure coding is specified. 
- How about a maximum just to prevent completely unreasonable requests. Perhaps: `maximum: 10`
- We also need these properties for erasure coding. See pool-ec.yaml for an example with `dataChunks` and `codingChunks`

Reply:
Thanks for the comment!.
Sure I'll look into `pool-ec.yaml`, my main aim behind this PR was your review.